tests/mcmc/test_sampling.py

```diff
@@ -223,10 +223,11 @@ def test_mala(self):
 
         mala = blackjax.mala(logposterior_fn, 1e-5)
         state = mala.init({"coefs": 1.0, "log_scale": 1.0})
-        _, states, _ = run_inference_algorithm(
+        _, states = run_inference_algorithm(
             rng_key=inference_key,
             initial_state=state,
             inference_algorithm=mala,
+            transform=lambda state, info: state,
```

Comment:
Let's simplify all the test in this file to `transform=lambda state, info: state.position,` if it applies.